Intersect (Core)/Configuration/ClientConfiguration.cs

```diff
@@ -162,6 +162,11 @@ public void Validate()
         /// </summary>
         public bool EnableContextMenus { get; set; } = true;
 
+        /// <summary>
+        /// Configures whether the highlighted even rows of list elements should be marked differently or not.
+        /// </summary>
+        public bool MarkListEvenRows { get; set; } = true;
```

Comment:
It took me a while to remember what this is actually called, but change the name to `EnableZebraStripedRows` (the name of the actual feature, but also made slightly more generic for use with tables).